language/move-lang/tests/move_check/parser/expr_return_missing_value.move

```diff
@@ -0,0 +1,6 @@
+module M {
+    f(v: u64) {
+        // Returns always require a value
```

Comment:
We should fix this, probably

Reply:
That is, a return without a value should be equivalent to `return ()`

Reply:
OK. I made a note to myself to look into that.